build/includes/website.mk

```diff
@@ -25,12 +25,14 @@
 #
 
 # generate the latest website
+site-server: HUGO_ENV ?=snapshot
```

Comment:
This does change how the development server works - which is usually to show the latest development version -- wondering if it should stay that way?

```suggestion
```

Also, wouldn't this need to be `HUGO_ENV=snapshot` ? But given that we already have an `ENV` var for passing in custom parameters, I think we have the use case of changing the HUGE_ENV environment variables covered?

Reply:
Good call, I can change it back. I mostly had this here for testing.

Reply:
Actually testing this without the HUGO_ENV the site-server default environment is "development" -- should I change this to site-server: HUGO_ENV=production?

Reply:
Leaving a note for my future self: The site-server is defaulting the environment to `development` because we are using an older version of Hugo. If we upgrade Hugo in the future the default environment behavior changes to `production` and we will need to explicitly call out the `development` environment.